architecture/proposed/long-term-plans.md

```diff
@@ -29,6 +29,12 @@ The design of the ALT interfaces is cumbersome, especially when replacing a whol
 
 In Mbed TLS 3.0, existing ALT implementations continue to work, since PSA drivers are not fully implemented yet. However, it is likely that no new ALT possibility will be added. Once PSA drivers are ready for production, ALT implementations will be deprecated, likely to be removed in Mbed TLS 4.0.
 
+### Automated and standardised code styling
+
+Our [coding standards](https://tls.mbed.org/kb/development/mbedtls-coding-standards) are currently enforced by manual code reviews which can lead to inconsistencies across the code base and places the burden on reviewers to spot these errors. It would be much more effective and efficient to automate the enforcement of the coding standards so that non-compliance is found and reported via an appropriate error.
```

Comment:
Another motivation is that the style is unusual, which is a barrier to entry. We're not just planning to make the style more consistent, but also more usual.

Reply:
Done.